src/content/docs/plugins/submission.md

```diff
@@ -2,3 +2,96 @@
 title: Plugin Submission
 description: How to submit your plugin to the official Vencord repository
 ---
+
+Submitting a plugin to Vencord allows you to share your work with the community and have it included in the official Vencord releases. Follow this guide to submit your plugin for consideration.
+
+## Prerequisites
+
+Before submitting your plugin, ensure that:
+- Your plugin is fully developed and tested (refer to the [Creating Plugins guide](/plugins/creating) for help).
+- You have a GitHub account and are familiar with basic Git commands.
+- Your plugin adheres to the code quality, structure, and naming conventions used in Vencord (e.g., camelCase for plugin folder names).
+
+## Submission Process
+
+The submission process involves forking the Vencord repository, adding your plugin, and creating a pull request for review by the maintainers.
+
+### Step 1: Fork the Vencord Repository
+
+1. Go to the official [Vencord GitHub repository](https://github.com/Vendicated/Vencord).
+2. Click the **Fork** button in the top-right corner to create a copy of the repository under your GitHub account.
+3. Clone the forked repository to your local machine:
+   ```bash
+   git clone https://github.com/YourUsername/Vencord
+   cd Vencord
+   ```
+
+### Step 2: Add Your Plugin
+
+1. Navigate to the `src/plugins` folder in your cloned repository:
+   ```bash
+   cd src/plugins
+   ```
+2. Create a new folder for your plugin using camelCase (e.g., `myFirstPlugin`):
+   ```bash
+   mkdir myFirstPlugin
+   ```
+3. Add your plugin files (`index.ts`, `README.md`, and optionally `native.ts`):
+    - **`index.ts`**: This is the main entry point for your plugin's code.
+    - **`README.md`**: Document your plugin, explain its functionality, and provide installation instructions and usage examples.
+    - **`native.ts`** (optional): If your plugin uses Node.js APIs, include this file.
+  
+   Make sure the `README.md` includes:
+   - A short description of your plugin.
+   - Instructions for installing and using the plugin.
+   - Screenshots, GIFs, or videos if applicable to showcase your plugin.
+
+### Step 3: Commit Your Changes
+
+1. Add your changes to git:
+   ```bash
+   git add src/plugins/myFirstPlugin
+   ```
+2. Commit your changes with a descriptive message:
+   ```bash
+   git commit -m "Add myFirstPlugin"
+   ```
+3. Push the changes to your forked repository:
+   ```bash
+   git push origin main
+   ```
+
+### Step 4: Create a Pull Request
+
+1. Go to your GitHub fork of the Vencord repository.
+2. Click on the **Pull Requests** tab and then click **New Pull Request**.
+3. Ensure that the base repository is `Vendicated/Vencord` and the base branch is `main`.
```

Comment:
base branch should be dev, if it isnt then maintainers will switch it to dev anyway